app/uk/gov/hmrc/taxenrolmentassignmentfrontend/views/ReportSuspiciousID.scala.html

```diff
@@ -0,0 +1,106 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.taxenrolmentassignmentfrontend.config.AppConfig
+@import uk.gov.hmrc.taxenrolmentassignmentfrontend.views.html.Layout
+@import uk.gov.hmrc.taxenrolmentassignmentfrontend.models.AccountDetails
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
+
+@this(
+layout: Layout,
+govukSummaryList : GovukSummaryList,
+govukDetails : GovukDetails,
+govukButton: GovukButton)
+
+@(currentAccountHasSA: Boolean = false)(implicit request: Request[_], messages: Messages)
+
+@createRowWith(key: String, value: String) =  {
+<div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+        @key
+    </dt>
+    <dd class="govuk-summary-list__value">
+        @value
+    </dd>
+</div>
+}
+
+@content = {
+<p class="govuk-body">@messages("reportSuspiciousId.detailsComponent.text1")</p>
+<p class="govuk-body">@messages("reportSuspiciousId.detailsComponent.text2")</p>
+<a href=#>@messages("reportSuspiciousId.detailsComponent.link")</a>
+<p>@messages("reportSuspiciousId.detailsComponent.text3")</p>
+<p class="govuk-body">@messages("reportSuspiciousId.detailsComponent.text4")</p>
+}
+
+
+@layout(
+pageTitle = Some(messages("reportSuspiciousId.title")), Some("?")) {
+
+<h1 class="govuk-heading-xl">@{messages(s"reportSuspiciousId.heading")}</h1>
+
+<dl class="govuk-summary-list">
+    @createRowWith(messages("reportSuspiciousId.userId"), "*********8173")
```

Comment:
personal information relating to the user should be passed in using the accountDetails model. For reference the the PTEnrolmentOnAnotherAccount is a similar page a shows details relating to a user.